util/ci/ubuntu-14.04-testscript

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+git clone --depth 1 https://github.com/Gnucash/gnucash.git;
+mkdir build;
+cd build;
+export TZ="America/Los_Angeles";
+ls -al /;
+ls -al /gtest;
+mkdir -p /root/.local/share
+export GTEST_ROOT=/gtest/googletest GMOCK_ROOT=/gtest/googlemock;
+if [[ "$BUILDTYPE" == "cmake-make" ]]; then
```

Comment:
What do you think about putting this 'if buildtype...' block in a separate file and calling it from the various distro test scripts?

Reply:
That will of course only work if the commands are exactly the same for all distros, but it's worth investigating. For instance, on arch, googletest is installed on the system path while on Ubuntu 14.04 we install it ourselves locally and hence have to pass GTEST_ROOT/GMOCK_ROOT.